app/helpers/bgeigie_imports_helper.rb

```diff
@@ -33,4 +33,16 @@ def operation_button(bgeigie_import, action, text = t(format('.%s', action)))
       f.submit text, class: 'btn btn-primary'
     end
   end
+
+  def unmoderated_id_list
+    unmoderated_id_list = []
+    @bgeigie_imports.each do |bgeigie_import|
+      unmoderated_id_list << bgeigie_import.id unless bgeigie_import.approved?
+    end
+    unmoderated_id_list
+  end
+
+  def moderator?
```

Comment:
```suggestion
  def moderator?(user)
```

This helper also should receive `User` object as argument.